### PR TITLE
add possibility to manage the version of the extra packages

### PR DESCRIPTION
**Pull request by cristifalcas**
We manage some of the packages externally and we would like to keep them updated. This PR offers the possibility to do that without conflicts.

### CODE REVIEW

**Comment by ekohl** [comment]
I don't think I like one parameter for all packages because then you can only use installed or latest, but specific versions will break badly. @mmoll what's your opinion?

**Comment by mmoll** [comment]
Yeah, that won't work for versions. @cristifalcas I guess your use case is to set it to `latest` instead of just `present`?

**Comment by cristifalcas** [comment]
Yes, we want some packages to be always updated.

**Comment by cristifalcas** [comment]
The other possibility is to add a new parameter for each package, that will look ugly :)

**Comment by mmoll** [comment]
maybe limit the choice to installed/present/latest?

**Comment by ekohl** [comment]
I could live with that.

**Comment by mmoll** [comment]
b7a

File: manifests/init.pp [hunk 1]
```diff
@@ -15,6 +15,9 @@
 # $version::                    foreman package version, it's passed to ensure parameter of package resource
 #                               can be set to specific version number, 'latest', 'present' etc.
 #
+# $ensure_packages_version::    control extra packages version, it's passed to ensure parameter of package resource
+#                               can be set to specific version number, 'latest', 'present' etc.
```
Comment (mmoll) [inline manifests/init.pp]: this comment needs updating now

**Comment by mmoll** [comment]
:+1:

**Comment by ekohl** [comment]
Thanks!

**Pull request closed without merging**